_data/compatibility/opendime.yaml

```diff
@@ -0,0 +1,58 @@
+---
+name: Opendime
+internal_url: /en/compatibility/opendime
+logo: /img/compatibility/opendime/opendime.png
+rbf:
+  tested:
+    date: "2018-10-18"
+    platforms:
+      - macOS
+    version: "2.1.0"
+  features:
+    receive:
+      notification: "na"
+      list: "false"
+      details: "na"
+      shows_replaced_version: "true"
+      shows_original_version: "false"
+    send:
+      signals_bip125: "false"
+      list: "false"
+      details: "na"
+      shows_replaced_version: "na"
+      shows_original_version: "na"
+  examples:
+    - image: /img/compatibility/opendime/sending-instructions.png
+      caption: >
+        Sending RBF Transaction - Opendime does not have send transaction capabilities. You need to sweep the private key to use in a regular wallet.
```

Comment:
Suggest removing the 'Sending RBF Transaction' since the software does not allow sending an RBF-signaling tx

Reply:
Agreed here. Brought it up in a discussion here for consensus: https://github.com/bitcoinops/bitcoinops.github.io/pull/183#issuecomment-519931177